providers/pool.rb

```diff
@@ -245,3 +270,11 @@ def configure
     end
   end
 end
+
+private
+def configure_application_pool(condition, config, add_remove = '')
+  if(condition)
+    @was_updated = true
+    @cmd << " \"/#{add_remove}[name='#{@new_resource.pool_name}'].#{config}\""
```

Comment:
The IIS docs keep mentioning the need for "/commit:apphost" for a number of commands.  I'm not sure of what the significance is.  Do we need to add that here?